app/src/main/java/com/versobit/kmark/xhangouts/Setting.java

```diff
@@ -46,6 +46,18 @@
     UI_SEND_LOCK,
     UI_DISABLE_PROXIMITY,
     UI_APP_COLOR,
+    UI_INCOMING("incomingColor"),
+    UI_INCOMING_OTR("incomingColorOTR"),
+    UI_INCOMING_FONT("incomingFontColor"),
+    UI_INCOMING_FONT_OTR("incomingFontColorOTR"),
+    UI_INCOMING_LINK("incomingLinkColor"),
+    UI_INCOMING_LINK_OTR("incomingLinkColorOTR"),
+    UI_OUTGOING("outgoingColor"),
+    UI_OUTGOING_OTR("outgoingColorOTR"),
+    UI_OUTGOING_FONT("outgoingFontColor"),
+    UI_OUTGOING_FONT_OTR("outgoingFontColorOTR"),
+    UI_OUTGOING_LINK("outgoingLinkColor"),
+    UI_OUTGOING_LINK_OTR("outgoingLinkColorOTR"),
```

Comment:
Can we change these to something more standard like the others? These arguably do not even need custom names. `UI_OUTGOING_LINK_OTR` for example would automatically become `ui_outgoing_link_otr` which is pretty uniform. We could also prefix the new constants with `UI_COLOR_` instead of just `UI_` or something similar so it's obvious they are all for this feature and not say the `UI_APP_COLOR`.